src/app/script/engine.cpp

```diff
@@ -474,6 +474,9 @@ Engine::Engine()
   setfield_integer(L, "LEFT",   ui::LEFT);
   setfield_integer(L, "CENTER", ui::CENTER);
   setfield_integer(L, "RIGHT",  ui::RIGHT);
+  setfield_integer(L, "TOP",    ui::TOP);
+  setfield_integer(L, "BOTTOM", ui::BOTTOM);
```

Comment:
I'm not a big fan of exposing these internal flags as they are, but it's the most simple way, we can keep this as is.